guide/interactions/modals.md

```diff
@@ -101,20 +100,42 @@ Restart your bot and invoke the `/ping` command again. You should see a popup fo
 
 Modals are received via an interaction. You can check if a given interaction is a modal by invoking the `#isModalSubmit()` method:
```

Comment:
this typeguard is being removed, use `interaction.type === InteractionType.ModalSubmit`